src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,61 @@
+package duke;
+
+import duke.commands.Command;
+import duke.exceptions.DukeException;
+import duke.parser.Parser;
+import duke.storage.Storage;
+import duke.tasklist.TaskList;
+import duke.ui.Ui;
+
+/**
+ * The Duke class is the main class which starts the chatbot
+ */
+public class Duke {
+
+    private TaskList taskList;
+    private final Ui ui;
+    private final Storage storage;
+    private final String FILEPATH = "./data/duke.txt";
+    private final String FOLDERPATH = "./data";
```

Comment:
Maybe you can use underscore to separate the words?
```suggestion
    private final String FILE_PATH = "./data/duke.txt";
    private final String FOLDER_PATH = "./data";
```